lib/libvmod_vtc/vmod_vtc.vcc

```diff
@@ -169,6 +169,20 @@ version ``v1``).
 Candidate for moving into vmod_proxy, but there were concerns about
 the interface design
 
+$Function DURATION benchmark(SUB, INT total=1000, INT leg=10)
```

Comment:
Just a naming comment, but IMO "leg" is a strange name here. Initially I thought it might be an obscure abbreviation for something like "legacy" or "legal", or maybe an equally obscure allusion to arms and legs. The docs didn't help me much in understanding (more about that further down), I had to read the code to get the idea.

Admittedly it's hard to think of compelling names for the parameters. What's happening here is inner iterations without rollbacks and outer iterations with rollbacks. But `inner_iters_without_rollback` and `outer_iters_with_rollback` are, well ...

Reply:
I had asked [my advisor the lion](https://dict.leo.org/german-english/Teilstrecke), if anyone has a better suggestion, please let me know. But two from the previous comment, are, well ... :)

Reply:
If we're looking to the lion, there are two suggestions there that IMO are better: stages, sections. If it were my choice, I'd go with stages. BTW I think plural would be better in any case -- it's a number of iterations, and more than one will almost always be chosen.

#bikeshed except that IMNSHO "leg" is the worst of these choices.